_includes/header.html

```diff
@@ -4,6 +4,17 @@
 
     {% assign custom_url = site.url | append: site.baseurl %}
     {% assign full_base_url = custom_url | default: site.github.url %}
+    {% if site.header_pages %}
```

Comment:
Could this be written as `{% assign header_pages = site.header_pages | default: site.pages %}`?